deploy

```diff
@@ -0,0 +1,3 @@
+rm /usr/bin/odyssey
```

Comment:
This script seems to lack extension and portability... do we really want it in main repo?

Reply:
Deleted